src/dashboard/Data/Browser/CreateClassDialog.react.js

```diff
@@ -67,9 +67,9 @@ export default class CreateClassDialog extends React.Component {
         title='Add a new class'
         subtitle='Create a new collection of objects.'
         disabled={!this.valid()}
-        confirmText='Create class'
+        confirmText='Add'
```

Comment:
Maybe let's change the terminology here - you "create a class" and you can "add columns to a class".


```
title='Create a new class?'
subtitle='This creates a new class to hold objects.'
confirmText='Create'
continueText={'Create & add columns'}
```

So we prevent the `Add & add columns` which sounds a bit strange.

Reply:
Yep, sounds weird.. 
Fixed!